modules/ibi-create-iso-for-bmh.adoc

```diff
@@ -0,0 +1,147 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-image-based-install.adoc 
+
+:_mod-docs-content-type: PROCEDURE
+[id="ibi-create-iso-for-bmh_{context}"]
+= Creating a live installation ISO for a {sno} image-based installation
+
+You can embed your {sno} seed image URL, and other installation artifacts, in a live installation ISO by using the `openshift-install` installation program. 
+
+[NOTE]
+====
+For more information about the specification for the `image-based-installation-config.yaml` manifest, see the section _Reference specifications for the `image-based-installation-config.yaml` manifest_.
+====
+
+.Prerequisites
+* You generated a seed image from a {sno} seed cluster.
+* You downloaded the latest version of the `openshift-install` installation program.
+* The target host has network access to the seed image URL and all other installation artifacts.
+* If you require static networking, you must install the `nmstatectl` library on the host that creates the live installation ISO.
+
+.Procedure
+
+. Create a live installation ISO and embed your {sno} seed image URL and other installation artifacts:
+
+.. Create a working directory by running the following:
++
+[source,terminal]
+----
+$ mkdir ibi-iso-workdir <1>
+----
+<1> Replace `ibi-iso-workdir` with the name of your working directory.
+
+.. Optional. Create an installation configuration template to use as a reference when configuring the `ImageBasedInstallationConfig` resource:
++
+[source,terminal]
+----
+$ openshift-install image-based create image-config-template --dir ibi-iso-workdir <1>
+----
+<1> If you do not specify a working directory, the command uses the current directory.
++
+.Example output
+[source,terminal]
+----
+INFO Image-Config-Template created in: ibi-iso-workdir
+----
++
+The command creates the `image-based-installation-config.yaml` installation configuration template in your target directory:
```

Comment:
Since we're providing a complete example in the next step, I don't think we need to add this here.